src/Process/Runner/InitialTestsRunner.php

```diff
@@ -47,9 +47,14 @@
  */
 final class InitialTestsRunner
 {
+    private const ERROR_TIMEOUT = 10;
```

Comment:
I think a comment would be appreciated here, at the very least to specify the unit of that number

Reply:
Updated